packages/manager/apps/pci-instances/src/adapters/tanstack/instances/right/mapper.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  TOperationStatus,
+  TInstanceCreationData,
+} from '@/domain/entities/instance';
+import { TCreateInstanceDTO, TCreateInstanceResponseDTO } from './dto.type';
+import { TOperationStatusDTO } from '../../operations/right/dto';
+
+type TCreateInstanceEntity = {
+  id: string;
+  name: string;
+  quantity: number;
+  availabilityZone: string | null;
+  imageId: string | null;
+  imageRegion: string | null;
+  existingSshKeyId: string | null;
+  newSshPublicKey: string | null;
+};
+
+export const mapFlavorToDTO = (
+  payload: TCreateInstanceEntity,
```

Comment:
```suggestion
  entity: TCreateInstanceEntity,
```

Payload means nothing ;)